src/edu/washington/escience/myria/storage/Tuple.java

```diff
@@ -157,7 +159,7 @@ public void putBoolean(final int column, final boolean value) {
   }
 
   @Override
-  public void putDateTime(final int column, final DateTime value) {
+  public void putDateTime(final int column, @Nonnull final DateTime value) {
```

Comment:
Simply because I didn't see them in the list of Java warnings when fixing `putDateTime` :) Fixed.